code/game/objects/items/reagent_containers/hypospray.dm

```diff
@@ -120,15 +120,8 @@
 		if(!do_after(user, SKILL_TASK_EASY, TRUE, 5, BUSY_ICON_BUILD))
 			return
 
-
-	var/list/injected = list()
-	for(var/datum/reagent/R in reagents.reagent_list)
-		injected += R.name
-	var/contained = english_list(injected)
-	log_combat(user, A, "injected", src, "Reagents: [english_list(reagents.reagent_list)]")
-
-	if(ismob(A))
-		var/mob/M = A
+	if(isliving(A))
```

Comment:
You already check for living a few lines up.

Reply:
That actually checks all other atoms but mobs.